src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetApiPackageManager.cs

```diff
@@ -173,54 +173,47 @@ private async Task<(PackageSource, IPackageSearchMetadata)> GetLatestVersionInte
             }
             _ = packageSources ?? throw new ArgumentNullException(nameof(packageSources));
 
+            (PackageSource source, IEnumerable<IPackageSearchMetadata> foundPackages)[] foundPackagesBySource =
+                await Task.WhenAll(
+                    packageSources.Select(source => GetPackageMetadataAsync(source, packageIdentifier, includePrerelease: true, cancellationToken)))
```

Comment:
`includePrerelease: true`, but this method has `includePreview` parameter, should it be used?

Reply:
Nevermind, read rest of method.